src/assets/i18n/en_US.json

```diff
@@ -229,111 +229,111 @@
+  "account.order_templates.edit_form.save_button.text": "Save changes",
   "account.orderdetails.add_to_cart.button.label": "Reorder",
   "account.orderdetails.approved.text": "(approved by {{0}} on {{1}})",
-  "account.orderdetails.breadcrumb": "Order Details",
-  "account.orderdetails.date_of_order.label": "Date of Order",
+  "account.orderdetails.breadcrumb": "Order details",
+  "account.orderdetails.date_of_order.label": "Date of order",
   "account.orderdetails.headerDetails.default": "Below you will find details about the item(s) in your order. If you ordered more than one item, please note that some items may display a different shipping method and/or status because they are shipped in a separate package.",
-  "account.orderdetails.heading.default": "Order Details",
-  "account.orderdetails.links.continue_shopping": "Continue Shopping",
-  "account.orderdetails.links.return_to_orders": "Back to Orders",
-  "account.orderdetails.order_number.label": "Order No.",
-  "account.orderdetails.order_status.label": "Order Status",
+  "account.orderdetails.heading.default": "Order details",
+  "account.orderdetails.links.continue_shopping": "Continue shopping",
+  "account.orderdetails.links.return_to_orders": "Back to orders",
+  "account.orderdetails.order_number.label": "Order no.",
+  "account.orderdetails.order_status.label": "Order status",
   "account.orderdetails.print_link.text": "Print",
   "account.orderlist.no_orders_message": "Currently there are no orders.",
   "account.orderlist.no_placed_orders_message": "You have not placed an order yet.",
   "account.orderlist.table.buyer": "Buyer",
-  "account.orderlist.table.date_of_order": "Order Date",
+  "account.orderlist.table.date_of_order": "Order date",
   "account.orderlist.table.destination": "Destination",
   "account.orderlist.table.items": "Items",
-  "account.orderlist.table.order_number": "Order No.",
+  "account.orderlist.table.order_number": "Order no.",
   "account.orderlist.table.order_status": "Status",
-  "account.orderlist.table.order_total": "Order Total",
-  "account.ordertemplates.heading": "Order Templates",
-  "account.ordertemplates.link": "Order Templates",
-  "account.ordertemplates.widget.heading": "My Order Templates",
-  "account.ordertemplates.widget.view_all.link": "View All Order Templates",
-  "account.organization.cost_center_management": "Cost Center Management",
-  "account.organization.cost_center_management.back_to_list": "Back to Cost Center Management",
+  "account.orderlist.table.order_total": "Order total",
+  "account.ordertemplates.heading": "Order templates",
+  "account.ordertemplates.link": "Order templates",
+  "account.ordertemplates.widget.heading": "My order templates",
+  "account.ordertemplates.widget.view_all.link": "View all order templates",
+  "account.organization.cost_center_management": "Cost center management",
+  "account.organization.cost_center_management.back_to_list": "Back to cost center management",
```

Comment:
Is Cost Center Management a proper name (because product name)?

Reply:
It's no product name. It's just the place where you can configure your cost centers. It's the name of a page or nav item.

Reply:
So page names should not be capitalized?

Reply:
We rarely use the product names in PWA. Maybe the ICM, if you have to configure something there as an admin. I don't think storefront customers need to know which systems are behind the store in general.